x-pack/test/functional/apps/ml/data_visualizer/index_data_visualizer.ts

```diff
@@ -283,22 +322,47 @@ export default function ({ getService }: FtrProviderContext) {
       await ml.dataVisualizerIndexBased.assertTimeRangeSelectorSectionExists();
 
       await ml.testExecution.logTestStep(`${testData.suiteTitle} loads data for full time range`);
-      await ml.dataVisualizerIndexBased.clickUseFullDataButton(testData.expected.totalDocCount);
+      await ml.dataVisualizerIndexBased.clickUseFullDataButton(
+        testData.expected.totalDocCountFormatted
+      );
 
       await ml.testExecution.logTestStep(`${testData.suiteTitle} displays the panels of fields`);
       await ml.dataVisualizerIndexBased.assertFieldsPanelsExist(testData.expected.fieldsPanelCount);
 
+      await ml.testExecution.logTestStep(`${testData.suiteTitle} displays the Metrics panel`);
+      await ml.dataVisualizerIndexBased.assertFieldsPanelForTypesExist([ML_JOB_FIELD_TYPES.NUMBER]);
+
+      await ml.testExecution.logTestStep(
+        `${testData.suiteTitle} displays the expected document count card`
+      );
+      await ml.dataVisualizerIndexBased.assertCardExists(
+        testData.expected.documentCountCard.type,
+        testData.expected.documentCountCard.fieldName
+      );
+      await ml.dataVisualizerIndexBased.assertDocumentCountCardContents();
+
+      await ml.testExecution.logTestStep(
+        `${testData.suiteTitle} displays the expected metric field cards and contents`
+      );
+
       if (testData.expected.metricCards !== undefined && testData.expected.metricCards.length > 0) {
         await ml.testExecution.logTestStep(`${testData.suiteTitle} displays the Metrics panel`);
         await ml.dataVisualizerIndexBased.assertFieldsPanelForTypesExist([
           ML_JOB_FIELD_TYPES.NUMBER,
```

Comment:
Do we need this check again here? I can see that you've added it a couple lines above.

Reply:
Good spot. Removed in https://github.com/elastic/kibana/pull/83174/commits/563f2a3749888bd3ea819800822f913d34b8291f